Gems/Multiplayer/Code/Include/Multiplayer/AutoGen/AutoComponent_Header.jinja

```diff
@@ -533,6 +533,7 @@ namespace {{ Component.attrib['Namespace'] }}
         bool HasController() const override;
         Multiplayer::MultiplayerController* GetController() override;
         const Multiplayer::MultiplayerController* GetController() const override;
+        static AZ::HashValue64 GetVersionHash() { return s_versionHash; }
```

Comment:
Really try to keep implementations out of headers.. even trivial guys like this, as it has a tendency to get out of control quickly in a large codebase like what we're dealing with..

Reply:
Thank you, fixed